metrics/wait.io.table/query.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Block, Inc.
+
+package waitiotable
+
+import (
+	"fmt"
+	"strings"
+)
+
+func TableIoQuery(set map[string]string, metrics []string) (string, error) {
+	columns := setColumns(set, metrics)
+
+	query := fmt.Sprintf("SELECT %s FROM performance_schema.table_io_waits_summary_by_table", strings.Join(columns, ", "))
+	var where string
+	if include := set[OPT_INCLUDE]; include != "" {
+		where = setWhere(strings.Split(set[OPT_INCLUDE], ","), true)
+	} else {
+		where = setWhere(strings.Split(set[OPT_EXCLUDE], ","), false)
+	}
+	return query + where, nil
+}
+
+func setColumns(set map[string]string, metrics []string) []string {
+	columns := []string{"OBJECT_SCHEMA", "OBJECT_NAME"}
+
+	if all, ok := set[OPT_ALL]; ok && strings.ToLower(all) == "yes" {
+		for _, name := range metric_names {
+			columns = append(columns, name)
+		}
+	} else {
+		// Default
+		for _, metric := range metrics {
+			metric = strings.ToLower(metric)
+
+			if _, ok := metric_map[metric]; ok {
```

Comment:
Should it be an error if the column doesn't exist?

Reply:
I thought about it, but an error would mean that the metric would not collect at all and (possibly?) the plan would not load. Having a typo in a metric name is not show-stopping worthy in my name. At best we log it but don't throw an error. Really `TableIoQuery` doesn't throw _any_ errors so I should clean up the function definition.

Reply:
Hm... yeah this is a part of Blip design that's not really nailed down: how are metric typos handled (since all/most metrics are explicitly listed)? We can leave this here as-is and deal with that question later.